### PR TITLE
Update HomeFragment.kt

### LINKED ISSUE #53: change ItemAdapter2 class name
The ItemAdapter2 is used for Displaying Video recylerlist, but it isn't named properly.It should be renamed to VideoRecyclerViewAdapter in the files ItemAdapter2.kt and HomeFragment.kt.
Reference PR #49.

### PR DESCRIPTION
**Pull request by himanshu007-creator**
## Description
This fixes #53 

## Checklist:

- [ ] I used the Reformat code option in android studio for the files I changed
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] The App is running Fine and as Expected In my Emulator or Physical Device
- [ ] I have checked my code and corrected any misspellings
- [ ] My changes didn't cause any issues for the other functionalities present before.
- [x] Requested **reviews** from atleast two **Reviewers**
- [ ] This PR is created From a branch which is upto date with the master branch

### CODE REVIEW

**Review by Amar-2003 — CHANGES_REQUESTED** [review]
Thanks for the PR, you need to also change the line 31 in the file HomeFragment.kt

**Comment by himanshu007-creator** [comment]
> Thanks for the PR, you need to also change the line 31 in the file HomeFragment.kt

done !

File: app/src/main/java/com/example/insights/HomeFragment.kt [hunk 1]
```diff
@@ -28,7 +28,7 @@ class HomeFragment : Fragment() {
                     for (change in value.documentChanges) {
                         if (change.type == DocumentChange.Type.ADDED) {
                             videoValues.add(change.document.data as HashMap<String, String>)
-                            val Itemadapter = ItemAdapter2(this, videoValues)
+                            val VideoRecyclerViewAdapter = VideoRecyclerViewAdapter(this, videoValues)
```
Comment (Amar-2003) [inline app/src/main/java/com/example/insights/HomeFragment.kt:31]: rename this val into video_recycler_view_adapter

File: app/src/main/java/com/example/insights/HomeFragment.kt [hunk 2]
```diff
@@ -28,7 +28,7 @@ class HomeFragment : Fragment() {
...
                             recyclerView2.adapter = Itemadapter
```
Comment (Amar-2003) [inline app/src/main/java/com/example/insights/HomeFragment.kt:32]: on the rightside is the value defined earlier, and this too should be updated

**Review by Amar-2003 — CHANGES_REQUESTED** [review]
Left a comment PTAL

File: app/src/main/java/com/example/insights/HomeFragment.kt (hunk 2 above)
Comment (himanshu007-creator) [reply app/src/main/java/com/example/insights/HomeFragment.kt:32]: done with the first change, but can you tell whats the second one?

Comment (Amar-2003) [reply app/src/main/java/com/example/insights/HomeFragment.kt:32]: The adapter class name has been changes , and in this line we are setting the adapter for the given recylerview, so this line too should be changed.

Comment (himanshu007-creator) [reply app/src/main/java/com/example/insights/HomeFragment.kt:32]: > The adapter class name has been changed , and in this line we are setting the adapter for the given recylerview, so this line too should be changed.

sir can you tell what exactly i have to change?
i have renamed val into video_recycler_view_adapter, in line 32 what exactly needs to be written in right side

Comment (Amar-2003) [reply app/src/main/java/com/example/insights/HomeFragment.kt:32]: ok

Comment (Amar-2003) [reply app/src/main/java/com/example/insights/HomeFragment.kt:32]: on line 32 the RHS should be video_recycler_view_adapter instead of Item adapter

**Comment by Amar-2003** [comment]
For more clarity check at the reference PR mentioned in the issue

**Comment by Amar-2003** [comment]
@himanshu007-creator , are doing this??

**Comment by himanshu007-creator** [comment]
> @himanshu007-creator , are doing this??

yes

**Comment by himanshu007-creator** [comment]
can you tell in which lines i need to change what😅

**Comment by Amar-2003** [comment]
> can you tell in which lines i need to change what

On line 32 change Itemadapter to video_recycler_view_adapter in the RHS

**Review by Amar-2003 — APPROVED** [review]
LGTM !

**Comment by Amar-2003** [comment]
Thanks for the PR @himanshu007-creator

**Pull request merged**